copy-this-template-directory/index.md

```diff
@@ -1,16 +1,924 @@
+
+![whole flow](/copy-this-template-directory/directus_flow_1_full_.png "whole flow")
+
+***
+
+- Trigger node - webhook:
+
+![trigger node](/copy-this-template-directory/directus_flow_1_01_.png "trigger node")
+
+
+***
+
+- Node 2 - "Webhook / Request URL"
+
+![request node](/copy-this-template-directory/directus_flow_1_02_.png "request node")
+
+URL is `{{$env.GCALENDARHOOKURL}}` - the actual value in the environment variable will be set after Google Apps Script is published.
+
+Request body:
+```js
+{
+  "headers": {{$trigger.headers}}
+}
+```
+
+> note that {{$trigger.headers}} is not quoted!
+
+After you save this Flow, copy the resulting webhook URL somewhere (the URL can be copied from the trigger node, when in view mode, not in edit mode).
+
+***
+
+&nbsp; 
+
+### Flow "Send delete event to Google Calendar"
+
+Processing of the Delete event is a bit different from processing of Create / Update (next flow). It should be set to blocking, cause we need to "intercept" the delete command and read item data - we need to know the id of the Google Calendar event, so we can send it to Published Google Apps Script.
+
+![whole flow](/copy-this-template-directory/directus_flow_2_full_.png "whole flow")
+
+***
+
+- Trigger node - Event Hook
+
+![trigger node](/copy-this-template-directory/directus_flow_2_01_.png "trigger node")
+
+make sure that you have the same config:
+
+Type - Filter(Blocking)
+
+Scope - items.delete
+
+Collections - a collection of your choice
+
+Response - data of last operation
+
+
+***
+
+- Node 2 - "Read data"
+
+![node 02](/copy-this-template-directory/directus_flow_2_02_.png "node 02")
+
+IDs edit raw value to:
+```js
+[
+    "{{$trigger.payload[0]}}"
+]
+```
+Query is empty
+
+
+***
+
+- Node 3 - "Webhook / Request URL"
+
+![node 03](/copy-this-template-directory/directus_flow_2_03_.png "node 03")
+
+URL is `{{$env.GCALENDARHOOKURL}}` - the actual value in the environment variable will be set after Google Apps Script is published.
+
+Method is Post
+
+Request body:
+```js
+{
+  "data": {{$last}},
+  "action": "delete",
+  "pass": "{{$env.GCALENDARHOOKSECRET}}"
+}
+
+```
+
+> note that {{$last}} is not quoted!
+
+***
+
+&nbsp; 
+ 
+### Flow "Send create/update event to Google Calendar"
+Processing of Create / Update is more complicated than Delete, cause after we send this event info, we might receive the ID of the Google Calendar Event that was created and we must update the current Directus item with this ID. This operation is not blocking. 
```

Comment:
```suggestion
Processing of Create / Update is more complicated than Delete. After we send the event information, we might receive the ID of the Google Calendar Event that was created and we must update the current Directus item with this ID. This operation is not blocking. 
```

Reply:
Alternatively case -> because, info -> information. Make sure to write full words.